Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM ghcr.io/linuxserver/baseimage-alpine:3.16
+
+# set version label
+ARG BUILD_DATE
+ARG VERSION
+LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DATE}"
+LABEL maintainer="nomandera,nemchik"
+
+# environment settings
+ENV S6_BEHAVIOUR_IF_STAGE2_FAILS=2
+
+RUN \
+  echo "**** install runtime packages ****" && \
+  apk add --no-cache --upgrade \
+    curl \
+    fail2ban \
+    jq \
+    nftables && \
+  echo "**** copy fail2ban confs to /defaults ****" && \
+  mkdir -p \
+    /tmp/fail2ban-confs \
+    /defaults/fail2ban/action.d \
+    /defaults/fail2ban/filter.d \
+    /defaults/fail2ban/jail.d && \
+  curl -o \
+    /tmp/fail2ban-confs.tar.gz -L \
+    "https://github.com/linuxserver/fail2ban-confs/tarball/master" && \
+  tar xf \
+    /tmp/fail2ban-confs.tar.gz -C \
+    /tmp/fail2ban-confs --strip-components=1 && \
```

Comment:
Can you not just untar directly into /defaults? It looks like you're copying 1:1 afterwards anyway.

Reply:
I don't want the `.github` folder, but I guess I could remove that after extracting. I went with being (maybe too) specific.

Reply:
I don't suppose it has any real impact on build speed because it's still got to copy the same files, it's just less obvious what's happening at a glance.

Reply:
Resolved in https://github.com/linuxserver/docker-fail2ban/pull/1/commits/6878c98e634358f351f8f1f41e54e96ca859fdb0